apple_stocks/java/src/com/stepan/Main.java

```diff
@@ -0,0 +1,125 @@
+package com.stepan;
+
+import java.util.Random;
+
+public class Main {
+
+    // Brute force max profit: 999829
+    // Fork join max profit:   999829
+    // Brute force time: 44138273
+    // Fork join time:   2036238
+    public static void main(String[] args) {
+        int[] stockPricesYesterday = new int[] {10, 7, 5, 8, 11, 9};
+        int maxProfitBruteForceTest = getMaxProfitBruteForce(stockPricesYesterday);
+        assertEquals(6, maxProfitBruteForceTest);
+        int maxProfitForkJoinTest = getMaxProfitForkJoin(stockPricesYesterday);
+        assertEquals(6, maxProfitForkJoinTest);
+
+        int[] randomStockPrices = generateRandomStockPrices(10_000);
```

Comment:
What was the reasoning behind using random stock prices, over specific test cases?